source/includes/_groups.md

```diff
@@ -0,0 +1,366 @@
+`title` <br> *string*      | The group title
+`group_id` <br> *string*   | The group identifier
+`user_count` <br> *number* | The count of users in the group
+`permissions` <br> *array* | A list of permissions granted to these group members
+`limits` <br> *object*     | An object holding the group's limits on this domain
+`quotas` <br> *object*     | An object holding the group's quotas on this domain
+
+## List groups
+
+> Definition
+
+```HTTP
+GET https://{DOMAIN_ID}.opendatasoft.com/api/management/v2/groups/
+```
+
+> Example request
+
+```shell
+curl https://yourdomain.opendatasoft.com/api/management/v2/groups/ \
+    -u username:password
+```
+
+> Example response
+
+```json
+[
+    {
+        "title": "My Users",
+        "group_id": "my-users",
+        "user_count": 1,
+        "permissions": [
+            "publish_dataset",
+            "edit_dataset"
+        ],
+        "limits": {
+            "max_records_by_dataset": 1000000,
+            "max_datasets": 500
+        },
+        "quotas": {
+            "limit": 1000,
+            "unit": "day"
+        }
+    },
+    {...},
+    {...}
+]
+```
+
+This endpoint lists all the groups linked to the domain.
+
+### Returns
+
+Returns a list of Group objects.
+
+## Create a group on the domain
+
+> Definition
+
+```HTTP
+POST https://{DOMAIN_ID}.opendatasoft.com/api/management/v2/groups/
+```
+
+> Example request
+
+```shell
+curl https://yourdomain.opendatasoft.com/api/management/v2/groups/ \
+    -X POST \
+    -u username:password \
+    -d '{"title": "An API group"}'
+```
+
+> Example response
+
+```json
+{
+    "user_count": 0,
+    "limits": {},
+    "title": "An API group",
+    "quotas": {},
+    "group_id": "an-api-group",
+    "permissions": []
+}
+```
+
+Creates a new empty group with a title on the domain.
+
+### Body
+
+Parameter | Description
+--------- | -----------
+`title` <br> *string* | The group title
+
+### Returns
+
+Returns the newly created Group object.
+
+## Retrieve a group
+
+> Definition
+
+```HTTP
+GET https://{DOMAIN_ID}.opendatasoft.com/api/management/v2/groups/<group_id>/
+```
+
+> Example request
+
+```shell
+curl https://yourdomain.opendatasoft.com/api/management/v2/groups/my-users/ \
+    -u username:password
+```
+
+> Example response
+
+```json
+{
+    "title": "My Users",
+    "group_id": "my-users",
+    "user_count": 1,
+    "permissions": [
+        "publish_dataset",
+        "edit_dataset"
+    ],
+    "limits": {
+        "max_records_by_dataset": 1000000,
+        "max_datasets": 500
+    },
+    "quotas": {
+        "limit": 1000,
+        "unit": "day"
+    }
+}
+```
+
+This endpoint retrieves the requested group.
+
+### Returns
+
+Returns a Group object.
+
+## Update a group
+
+> Definition
+
+```HTTP
+PUT https://{DOMAIN_ID}.opendatasoft.com/api/management/v2/groups/<group_id>/
+```
+
+> Example request
+
+```shell
+curl https://yourdomain.opendatasoft.com/api/management/v2/groups/my-users/ \
+    -X PUT \
+    -u username:password \
```

Comment:
Just thinking about it, and it's not for this PR, but I think we'll need to change these to encourage using API keys instead of basic auth